docs-web/configuration/policy-resource.md

```diff
@@ -109,6 +117,78 @@ Referencing both allow and deny policies, as shown in the example below, is not
 - name: allow-policy-two
 ```
 
+### RateLimit
+
+The rate limit policy configures NGINX to limit the processing rate of requests.
+
+For example, the following policy will limit all subsequent requests coming from a single IP address once a rate of 10 requests per second is exceeded:
+```yaml
+rateLimit:
+  rate: 10r/s
+  zoneSize: 10M
+  key: ${binary_remote_addr}
+```
+
+> Note: The feature is implemented using the NGINX [ngx_http_limit_req_module](https://nginx.org/en/docs/http/ngx_http_limit_req_module.html).
+
+```eval_rst
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Description
+     - Type
+     - Required
+   * - ``rate``
+     - The rate of requests permitted. The rate is specified in requests per second (r/s) or requests per minute (r/m).
+     - ``string``
+     - Yes
+   * - ``key``
+     - The key which the rate limit is applied. Can contain text, variables, or a combination of them. Variables must be surrounded by ``${}``. For example: ``${binary_remote_addr}``. Accepted variables are ``$binary_remote_addr``, ``$request_uri``, ``$url``, ``$http_``, ``$args``, ``$arg_``, ``$cookie_``.
+     - ``string``
+     - Yes
+   * - ``zoneSize``
+     - Size of the shared memory zone. Only positive values are allowed. Allowed suffixes are ``k`` or ``m``, if none are present ``k`` is assumed.
+     - ``string``
+     - Yes
+   * - ``delay``
+     - The delay parameter specifies a limit at which excessive requests become delayed. If not set all excessive requests are delayed.
+     - ``int``
+     - No*
+   * - ``noDelay``
+     - Disables the delaying of excessive requests while requests are being limited. Overrides ``delay`` if both are set.
+     - ``bool``
+     - No*
+   * - ``burst``
+     - Excessive requests are delayed until their number exceeds the ``burst`` size in which case the request is terminated with an error.
+     - ``int``
+     - No*
+   * - ``dryRun``
+     - Enables the dry run mode. In this mode, requests processing rate is not limited, however, in the shared memory zone, the number of excessive requests is accounted as usual.
```

Comment:
```suggestion
     - Enables the dry run mode. In this mode, the rate limit is not actually applied, but the the number of excessive requests is accounted as usual in the shared memory zone.
```

I'm not sure what this actually means. Are we saying that the rate limit isn't enforced, but we will track what requests would have been affected if it were?